apps/server/src/modules/board/controller/api-test/board-create.api.spec.ts

```diff
@@ -63,6 +63,70 @@ describe(`create board (api)`, () => {
 				const dbResult = await em.findOneOrFail(ColumnBoardNode, boardId);
 				expect(dbResult.title).toEqual(title);
 			});
+
+			describe('Board layout', () => {
+				describe('When layout is omitted', () => {
+					it('should create a column board', async () => {
+						const { loggedInClient, course } = await setup();
+						const title = 'new board';
+
+						const response = await loggedInClient.post(undefined, {
+							title,
+							parentId: course.id,
+							parentType: BoardExternalReferenceType.Course,
+						});
+
+						const boardId = (response.body as { id: string }).id;
```

Comment:
did you guys talk about the idea to always set the layout property without having this default? I find it more "concrete" if we need to specificly decide on one or the other

Reply:
Right, we can be more precise by not having a default. Changed that.